scheduler/lambdoService.go

```diff
@@ -0,0 +1,74 @@
+package scheduler
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+type LambdoService struct {
+	URL 		string
+	MinioURL 	string
+}
+
+type LambdoSpawnRequest struct {
+	RootfsURL string `json:"rootfs"`
+}
+
+type LambdoSpawnResponse struct {
+	ID		string	`json:"ID"`
+	Port	uint16 	`json:"port"`
+	Address string	`json:"address"`
+}
+
+func (service *LambdoService) SpawnVM(function_id uuid.UUID) (data LambdoSpawnResponse, err error) {
+	var res *http.Response
+	defer func() {
+		if res != nil {
+			res.Body.Close()
+		}
+	}()
+
+	body, err := json.Marshal(&LambdoSpawnRequest{
+		RootfsURL: fmt.Sprintf("%s/%s", service.MinioURL, function_id),
+	})
+
+	if err != nil {
+		return 
+	}
```

Comment:
Maybe is there is an error we want to throw something again, it seems weird to me to just return

Reply:
It will return the error stored in `err`, that's Golang's magick for ya

Reply:
Ho ok, I didn't know, nice